src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -166,15 +166,12 @@
 
     // text area from getting updated. This is used in cases where the source
     // couldn't be parsed, and the user is given the option to edit it.
-    private boolean lastSourceAccepted = true; // This indicates whether the last
-
-    // attempt
+    private boolean lastFieldAccepted = true;
+    private boolean lastSourceAccepted = true; // This indicates whether the last attempt
     // at parsing the source was successful. It is used to determine whether the
```

Comment:
I don't know to which field the comment belongs. It is not my comment. Parts of the comment seem to be fragmented. Can you help?